.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,27 @@
+name: clang-format
+
+on: 
+  [push, pull_request]
+
+jobs:
+  clang-format:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup environment
+      run: | 
+        sudo apt update
+        sudo apt install clang-format-9 --yes
```

Comment:
is it really required to specify the version? I would prefer to stick with the default ubuntu one

Reply:
`ubuntu-latest` use v6 by default (` clang-format version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)`)